Applications/IRCClient/IRCAppWindow.cpp

```diff
@@ -97,23 +97,23 @@ void IRCAppWindow::setup_actions()
         m_client.handle_part_action(window->channel().name());
     });
 
-    m_whois_action = GAction::create("Whois user", GraphicsBitmap::load_from_file("/res/icons/16x16/irc-whois.png"), [&](auto&) {
+    m_whois_action = GAction::create("Whois user", { Mod_Ctrl, Key_W }, GraphicsBitmap::load_from_file("/res/icons/16x16/irc-whois.png"), [&](auto&) {
         auto input_box = GInputBox::construct("Enter nickname:", "IRC WHOIS lookup", this);
         if (input_box->exec() == GInputBox::ExecOK && !input_box->text_value().is_empty())
             m_client.handle_whois_action(input_box->text_value());
     });
 
-    m_open_query_action = GAction::create("Open query", GraphicsBitmap::load_from_file("/res/icons/16x16/irc-open-query.png"), [&](auto&) {
+    m_open_query_action = GAction::create("Open query", { Mod_Ctrl, Key_O }, GraphicsBitmap::load_from_file("/res/icons/16x16/irc-open-query.png"), [&](auto&) {
         auto input_box = GInputBox::construct("Enter nickname:", "Open IRC query with...", this);
         if (input_box->exec() == GInputBox::ExecOK && !input_box->text_value().is_empty())
             m_client.handle_open_query_action(input_box->text_value());
     });
 
-    m_close_query_action = GAction::create("Close query", GraphicsBitmap::load_from_file("/res/icons/16x16/irc-close-query.png"), [](auto&) {
+    m_close_query_action = GAction::create("Close query", { Mod_Ctrl, Key_C }, GraphicsBitmap::load_from_file("/res/icons/16x16/irc-close-query.png"), [](auto&) {
```

Comment:
Duh, how did I not realize that one... even copying from the IRC message box and such would be negatively affected by this!
I shall remove this and probably the bind for "Open query" too because `/query` command exists.